contents/docs/web-analytics/getting-started.mdx

```diff
@@ -0,0 +1,204 @@
+  alt="Referrers"
+  classes="rounded"
+/>
+
+#### UTM parameters
+
+UTM parameters are another way to get information about where people are coming from. By providing some structured values at the end of our URLs, we can segment visitors into groups.
+
+For example, if you advertise a sale on Google, you can use the URL to indicate the `utm_source` and `utm_campaign`:
+
+`posthog.com?utm_source=google,utm_campaign=summer-sale`
+
+Web analytics enables you to filter by all the various UTM parameters that are available, and you can learn more about them in our [UTM segmentation](/docs/data/utm-segmentation) doc. We also have a tutorial on [how to track performance marketing](/tutorials/performance-marketing) that is highly relevant.
+
+#### 📋 Use case: Tracking visitors from a sponsored video
+
+One example of how PostHog uses these features was a recent sponsored video on YouTube. We sponsored a [Fireship](https://www.youtube.com/c/Fireship) video, and we can see how it worked below.
+
+For the YouTube video, there's a link in the description for people that are interested in seeing what PostHog is, and it looks like this:
+
+`https://posthog.com/fireship`
+
+That URL gets transformed into the following UTM paramaters:
+
+`https://posthog.com/?utm_source=fireship&utm_campaign=fireship`
+
+Then, we can see how many visitors came from that video by using the "UTM source" option on our web analytics dashboard:
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/utm_source_light_21fa9c00fe.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/utm_source_dark_eada0f8178.png"
+  alt="UTM sources"
+  classes="rounded"
+/>
+
+### Demographic information
+
+Based on how users are accessing your website, web analytics also provides additional demographic metrics that can help answer the following:
+
+- What kinds of devices are visitors using? Laptops? Tablets? Phones?
+- Which browsers and operating systems are most popular?
+- Where are users located geographically? Which countries? Which cities?
+- What are the most popular languages for visitors?
+
+### Goals
+
+The goals section of the web analytics dashboard is all about tracking conversion goals.
+
+> **💡 Concept:** What is a conversion? A conversion is a key event or action you want users to do. If a user converted, it means they not only visited your website, but also performed some key action like signing up or subscribing.
+
+You can click the **Add conversion goal** button at the top of the web analytics dashboard, and check out our detailed guide on how to set up [conversion goals](/docs/web-analytics/conversion-goals).
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_11_04_at_12_29_08_3bba14e7e9.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_11_04_at_12_28_25_689e6c893d.png"
+  alt="Conversion goal"
+  classes="rounded"
+/>
+
```

Comment:
Maybe add a link to the conversion goals docs page here?

Reply:
Fixed in https://github.com/PostHog/posthog.com/pull/10053/commits/9f3700f8849606c85707b80c9e1ac8a3a9ee6282